cxx/non-atomic-action-interpreter-module/test/tests.cmake

```
@@ -0,0 +1,15 @@
+file(GLOB NON_ATOMIC_ACTION_INTERPRETER_TEST_AGENTS
+		"${CMAKE_CURRENT_LIST_DIR}/agent/*.cpp" "${CMAKE_CURRENT_LIST_DIR}/agent/*.hpp"
+		"${CMAKE_CURRENT_LIST_DIR}/keynodes/*.cpp" "${CMAKE_CURRENT_LIST_DIR}/keynodes/*.hpp")
+add_library(non_atomic_action_interpreter_test_agents STATIC ${NON_ATOMIC_ACTION_INTERPRETER_TEST_AGENTS})
+target_link_libraries(
+    non_atomic_action_interpreter_test_agents
+    non_atomic_action_interpreter
+    sc-agents-common)
+
+make_tests_from_folder(${CMAKE_CURRENT_LIST_DIR}/units
+    NAME non_atomic_action_interpreter_test_starter
+    DEPENDS non_atomic_action_interpreter_test_agents sc-builder-lib sc-core
```

```suggestion
    DEPENDS non_atomic_action_interpreter_test_agents sc-builder-lib
```
this one is not required

---

Linker error
[build] /usr/bin/ld: platform-dependent-components/problem-solver/reusable-components/cxx/non-atomic-action-interpreter-module/CMakeFiles/non_atomic_action_interpreter_test_starter.dir/test/units/testNonAtomicActionInterpreterAgent.cpp.o: undefined reference to symbol 'sc_memory_params_clear'
[build] /usr/bin/ld: /home/smn/ostis/ostis-metasystem/build/../bin/libsc-core.so: error adding symbols: DSO missing from command line

The metasystem does not include sc-machine, only its modules, it only works with this dependency.